db/views/searches_v07.sql

```diff
@@ -22,6 +22,7 @@ SELECT
   app_ver.application -> 'firm_office' ->> 'name' as firm_name,
   app_ver.application -> 'firm_office' ->> 'account_number' as account_number,
   app_ver.application ->> 'service_name' as service_name,
+  (app_ver.application -> 'cost_summary' ->> 'high_value')::boolean as high_value,
```

Comment:
Let's remove this line - you don't want to be modifying a previous version definition of the view